routes/web.php

```diff
@@ -46,9 +47,13 @@
         Route::post("/run-importers", [CityProviderController::class, "runImporters"]);
         Route::delete("/delete-city-without-assigned-country/{city}", [CityWithoutAssignedCountryController::class, "destroy"]);
         Route::post("/delete-all-cities-without-assigned-country", [CityWithoutAssignedCountryController::class, "destroyAll"]);
+
+        Route::get("/import-rules/{force}", [OpenAIService::class, "importRulesForAllCities"]);
```

Comment:
I think for being consistent we should move that method to `RulesController.php` and call that in it.